src/main/java/frc/robot/Constants.java

```diff
@@ -7,4 +7,10 @@
         public static final double POWER = 0.5; // [%]
     }
 
+    public static class Intake {
+        public static final double POWER = 0.5;
+        public static final double POWER_SPEED_RATIO = 0.5;
```

Comment:
Add units here too it may be helpful to understand the order is it %/(m/s) or the opposite